modules/social_features/social_event/src/ContentTranslationDefaultsConfigOverride.php

```diff
@@ -63,6 +138,31 @@ protected function getTranslationOverrides() {
         ],
         'translatable' => TRUE,
       ],
+      'views.view.upcoming_events' => [
+        'display' => [
+          'block_community_events' => $views_plugin_content_translation_settings,
+          'block_my_upcoming_events' => $views_plugin_content_translation_settings,
+          'page_community_events' => $views_plugin_content_translation_settings,
+          'upcoming_events_group' => $views_plugin_content_translation_settings,
+        ],
+      ],
+      'views.view.events' => [
+        'display' => [
+          'events_overview' => $views_plugin_content_translation_settings,
+          'block_events_on_profile' => $views_plugin_content_translation_settings,
+        ],
+      ],
+      'views.view.group_events' => [
+        'display' => [
+          'page_group_events' => $views_plugin_content_translation_settings,
+        ],
+      ],
+      'views.view.event_manage_enrollments' => $default_display_langcode_filter_options,
```

Comment:
@nechai As we discussed - could you check if we can avoid using config overrides, and just set needed values for translations by default?

Reply:
@ribel I tested and seems like we can add these changes directly to the views configurations. I added duplicated PR - https://github.com/goalgorilla/open_social/pull/2379